website/docs/docs/cloud/secure/databricks-privatelink.md

```diff
@@ -41,6 +41,23 @@ If using an existing Databricks workspace, all workloads running in the workspac
 
 :::
 
+## Configure Azure Private Link
+
+1. Navigate to your Azure Databricks workspace. 
+    The path is: `/subscriptions/<subscription_uuid>/resourceGroups/<resource_group_name>/providers/Microsoft.Databricks/workspaces/<workspace_name>`.
+2. From the workspace overview, click **JSON view**. 
+3. Copy the value in the `resource_id` field.  
+4. Add the required information to the template below, and submit your request to [dbt Support](https://docs.getdbt.com/community/resources/getting-help#dbt-cloud-support): 
+    ```
+      Subject: New Azure Multi-Tenant Private Link Request
+    - Type: Databricks
+    - Databricks instance name:
+    - Databricks Azure resource ID:
+    - dbt Cloud multi-tenant environment: EMEA
+    ```
+5. Once Support confirms the resources are available in the Azure portal, navigate to the Azure Databricks Workspace and go to **Networking** --> **Private Endpoint Connections** --> highlight the `dbt` named option, and select **Approve**.
```

Comment:
```suggestion
5. Once our support team confirms the resources are available in the Azure portal, navigate to the Azure Databricks Workspace and browse to **Networking** > **Private Endpoint Connections**. Then, highlight the `dbt` named option and select **Approve**.
```

Reply:
We treat "Support" like a proper noun in other places so I'll leave this for now to remain consistent. We change them in the future if folks feel one way or another about it

Reply:
to be fair, i've seen both in our docs. i received feedback in the past to use lower case like "sales team" / "support team" . 

totally non-blocking